Project Proposal (#1)
cynthiarich/whats-for-dinner: planning/project-proposal.md

> # What's for Dinner

Reviewer: ```suggestion
# What's for Dinner
```

**Description:** What do you want for dinner? It's a simple question that creates profound fear and dread -- what sounds good? What will taste good AND meet my dietary requirements? What's for Dinner? takes the stress out of mealtime planning by helping you plan a week at a time and even puts together your food order for you!

> **User Story** As a busy, working adult with special dietary needs, I need to take the stress out of mealtime planning so I can plan and prepare healthier meals for myself or my family.

Author: ```suggestion
**User Story** As a busy, working adult with special dietary needs, I need help to plan and prepare healthier meals for myself or my family and take the stress out of mealtime planning.
```
